src/custom/state/orders/utils.ts

```diff
@@ -118,20 +118,25 @@ type GetExecutionPriceParams = Omit<GetLimitPriceParams, 'buyAmount' | 'sellAmou
  */
 export function getExecutionPrice({
   executedBuyAmount,
-  executedSellAmount,
+  executedSellAmountBeforeFees,
   buyTokenDecimals,
   sellTokenDecimals,
   inverted,
 }: GetExecutionPriceParams): BigNumber {
   // Only calculate the price when both values are set
   // Having only one value > 0 is anyway an invalid state
-  if (!executedBuyAmount || !executedSellAmount || executedBuyAmount === '0' || executedSellAmount === '0') {
+  if (
+    !executedBuyAmount ||
+    !executedSellAmountBeforeFees ||
+    executedBuyAmount === '0' ||
+    executedSellAmountBeforeFees === '0'
+  ) {
     return ZERO_BIG_NUMBER
```

Comment:
In those cases the price should be null/undefined, not zero.

Reply:
As discussed in the chat, this is just mimicking https://github.com/gnosis/gp-ui/blob/36eae4570aac573e71d909042d3d4b18adfbf5aa/src/utils/operator.ts#L207-L207 to prevent division by zero